sdk/communication/communication-phone-numbers/README.md

```diff
@@ -45,36 +47,36 @@ You can get a key and/or connection string from your Communication Services reso
 
 Once you have a key, you can authenticate the `PhoneNumbersClient` with any of the following methods:
 
-### Create `KeyCredential` with `AzureKeyCredential`
+### Using a connection string
 
 ```typescript
-import { AzureKeyCredential } from "@azure/core-auth";
 import { PhoneNumbersClient } from "@azure/communication-phone-numbers";
 
-const credential = new AzureKeyCredential(KEY);
-const client = new PhoneNumbersClient(endpoint, credential);
+const connectionString = `endpoint=HOST;accessKey=KEY`;
+const client = new PhoneNumbersClient(connectionString);
 ```
 
-If you use a key to initialize the client you will also need to provide the appropriate endpoint. You can get this endpoint from your Communication Services resource in [Azure Portal][azure_portal].
+### Using an access key with `AzureKeyCredential`
 
-### Using a connection string
+If you use a key to initialize the client you will also need to provide the appropriate endpoint. You can get this endpoint from your Communication Services resource in [Azure Portal][azure_portal]. Once you have a key and endpoint, you can authenticate with the following code:
 
 ```typescript
+import { AzureKeyCredential } from "@azure/core-auth";
 import { PhoneNumbersClient } from "@azure/communication-phone-numbers";
 
-const connectionString = `endpoint=HOST;accessKey=KEY`;
-const client = new PhoneNumbersClient(connectionString);
+const credential = new AzureKeyCredential(KEY);
```

Comment:
Should we do something to help imply `KEY` is a string

Reply:
Something like..
```suggestion
const credential = new AzureKeyCredential("<KEY-from-your-resource>");
```